### PR TITLE
Fix circular import

### PR DESCRIPTION
**Pull request by Cub11k**
## Description
Fix for issue #1864
Added __all__ to util.py - for sphinx to generate docs accordingly

Extras:
Update README - add ru docs link

## Describe your tests
How did you test your change?

Python version:

OS:

## Checklist:
- [ ] I added/edited example on new feature/change (if exists)
- [ ] My changes won't break backward compatibility
- [ ] I made changes both for sync and async

### CODE REVIEW

**Comment by Cub11k** [comment]
I see that my pep8 formatter made some changes, this wasn't intentional, sorry xD

**Comment by coder2020official** [comment]
Please next time focus on one thing per PR.
Thoughts @Badiboy ?

**Comment by Badiboy** [comment]
Elegant enough except unit name :) I propose something like "library_util" or "service_functions"...

**Comment by Cub11k** [comment]
@Badiboy maybe "_service" looks good?

**Comment by Badiboy** [comment]
@Cub11k For me personally, I do not like names like _util or _service... I prefer full qualified names, but it's very subjective. If you and @coder2020official like "_util" or "_service" - let it be. It was just my personal oppinion for naming.

**Comment by Cub11k** [comment]
Renamed to "service_utils", looks fine for me

**Comment by Badiboy** [comment]
@Cub11k  👌

The last thing:

image

What for?

**Comment by Cub11k** [comment]
@Badiboy It's for sphinx to generate docs for methods, that are in service_utils.py now

Tried without _ _all_ _ - didn't work, maybe it requires some change it util.rst, but I'm not very familiar with it's behaviour

**Comment by Badiboy** [comment]
Let it be :)

File: telebot/service_utils.py [hunk 1]
```diff
@@ -0,0 +1,85 @@
+import random
+import string
+
+try:
+    # noinspection PyPackageRequirements
+    from PIL import Image
+    from io import BytesIO
+
+    pil_imported = True
+except:
```
Comment (Tishka17) [inline telebot/service_utils.py:10]: It should be `except ImportError` ?

File: telebot/service_utils.py [hunk 2]
```diff
@@ -0,0 +1,85 @@
+import random
+import string
+
+try:
+    # noinspection PyPackageRequirements
+    from PIL import Image
+    from io import BytesIO
```
Comment (Tishka17) [inline telebot/service_utils.py:7]: No ideas why problems wirh this import are expected and threaten as no PIL installed.

Probably this import should be done outside of try block

Comment (Cub11k) [reply telebot/service_utils.py:7]: PIL is not enlisted in package requirements, so it's the same as with redis for StateRedisStorage

File: telebot/service_utils.py (hunk 1 above)
Comment (Cub11k) [reply telebot/service_utils.py:10]: Agree, it definitely should be
That's why ctrl+C ctrl+V is bad xD

**Pull request merged**